lib/send_reset_email.rb

```diff
@@ -0,0 +1,27 @@
+require 'mailgun'
+
+class SendResetEmail
+
+  def initialize(user)
+    @user = user
+    @client = Mailgun::Client.new(ENV['MAILGUN_API_KEY'])
+  end
+
+  # def call
+  #   @client.send_message(to: @user.email, message: "You have requested a password reset. Follow this link to continue:
+  #     http://www.chitter.com/password_reset/#{@user.password_token}")
+  # end
+
+  def send_message
+    RestClient::Request.execute(
+      :url => "https://api:#{ENV['MAILGUN_API_KEY']}@api.mailgun.net/v3/sandbox119f7055446a45519d466456e9412df8.mailgun.org/messages",
```

Comment:
Metrics/LineLength: Line is too long. [134/90]<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.